package/firmware/ipq-wifi/Makefile

```diff
@@ -109,5 +110,6 @@ $(eval $(call generate-ipq-wifi-package,ezviz_cs-w3-wd1200g-eup,EZVIZ CS-W3-WD12
 $(eval $(call generate-ipq-wifi-package,linksys_ea8300,Linksys EA8300))
 $(eval $(call generate-ipq-wifi-package,mobipromo_cm520-79f,MobiPromo CM520-79F))
 $(eval $(call generate-ipq-wifi-package,qxwlan_e2600ac,Qxwlan E2600AC))
+$(eval $(call generate-ipq-wifi-package,wallystech_dr40x9,Wallystech DR40X9))
```

Comment:
Is this already submitted to the ath10k mailing list?

Reply:
No, but I gave up on submitting stuff as I have BDF-s that are now over 1-year old waiting to be merged.

Reply:
This happens very irregularly, i admit that. But please submit it so we can eventually drop the bdfs from our tree.

Reply:
Ok, will submit them.